.github/workflows/tests.yml

```diff
@@ -31,21 +29,18 @@ jobs:
           --health-timeout 5s
           --health-retries 5
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 100
 
-      - uses: actions/cache@v2
+      - uses: actions/cache@v4
         with:
           path: ~/.npm
           key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
       
       - name: npm install
         run: npm ci
 
-      - name: npm lint
-        run: npm run lint
-
```

Comment:
would be nice to keep this somewhere

Reply:
As mentioned above, I removed it because linting now also happens in the [Code Check CI](https://github.com/unit214/superhero-dex-backend/actions/runs/8357091913/job/22875539466) CI (At least npm run lint:check. npm run lint also fixes potential errors, I don't see why this is needed in the tests).

Reply:
was on the wrong branch :facepalm: